common/models/access-token.js

```diff
@@ -146,6 +146,16 @@ module.exports = function(AccessToken) {
       var userRelation = AccessToken.relations.user; // may not be set up
       var User = userRelation && userRelation.modelTo;
```

Comment:
> although: if I correctly get the docs for Registry.getModelByType(...) we could use getModelByType('User') instead of AccessToken.relations.user.modelTo as the method allows to
> to find configured models in models.json over the base model.

That may work in most cases, but I would be concerned about changes in edge cases, e.g. when the app has a custom User-based model that's actually not used for authentication, i.e. AccessToken is attached to the base User model. (That's just an example.)

Let's play it safe and keep the current implementation.